app/root.jsx

```diff
@@ -29,6 +29,18 @@ export default function App() {
         <meta charSet="utf-8" />
         <meta name="viewport" content="width=device-width,initial-scale=1" />
         <script defer data-domain="getalby.com" src="https://squirrel.getalby.com/js/plausible.js"></script>
+        <meta name="title" content="Alby — Lightning buzz for your Browser!"/>
+        <meta name="description" content="Alby brings Bitcoin to the web with in-browser payments and identity."/>
+        <meta property="og:type" content="website"/>
+        <meta property="og:url" content="https://getalby.com/"/>
+        <meta property="og:title" content="Alby — Lightning buzz for your Browser!"/>
+        <meta property="og:description" content="Alby brings Bitcoin to the web with in-browser payments and identity."/>
+        <meta property="og:image" content="/public/images/alby_icon_head_icon.png"/>
+        <meta property="twitter:card" content="summary_large_image"/>
+        <meta property="twitter:url" content="https://getalby.com/"/>
+        <meta property="twitter:title" content="Alby — Lightning buzz for your Browser!"/>
+        <meta property="twitter:description" content="Alby brings Bitcoin to the web with in-browser payments and identity."/>
+        <meta property="twitter:image" content="/public/images/alby_icon_head_icon.png"/>
```

Comment:
are these URLs correct or does remix also need some helper to create the proper URLs?

Reply:
Well they are correct in the project directory, is there a way to test without merging? I think we will have to merge and then test sharing functionality to see if it worked. No way to test the sharing without a real URL I think.

Reply:
you can try to access the URL locally and check the code how other images URLs are created.

Reply:
for example for the user images we do it like this: 
https://github.com/getAlby/website/blob/main/app/components/Header.jsx#L44-L45

import the image and dynamically set the HTML attribute

Reply:
thanks will check it out

Reply:
You are right, the URLs dont work in the browser. I have updated them to use the same React style src imports as the others. I think we could merge and test this now?